### DISCUSSION TITLE
请问NetGear R7800安装的OpenWRT，应该安装哪个版本？

### DISCUSSION THREAD

**Discussion by askiter** [messense/aliyundrive-webdav#300] [Q&A] [answered]
root@OpenWrt:~# opkg print-architecture
arch all 1
arch noarch 1
arch arm_cortex-a15_neon-vfpv4 10

下面这两个看着比较像，我试过不行
aliyundrive-webdav_1.2.4-1_arm_cortex-a5_vfpv4.ipk
aliyundrive-webdav_1.2.4-1_arm_cortex-a7_neon-vfpv4.ipk

谢谢

**Comment by messense** [comment, marked as answer]
`/etc/opkg.conf` 里面增加配置

```
arch all 1
arch noarch 1
arch arm_cortex-a15_neon-vfpv4 10
arch arm_cortex-a7_neon-vfpv4 100
```

然后再尝试安装 aliyundrive-webdav_1.2.4-1_arm_cortex-a7_neon-vfpv4.ipk

**Comment by askiter** [comment]
感谢，上面那个包安装上了，但是luci包又出错了

Executing package manager
Installing luci-app-aliyundrive-webdav (1.2.4) to root...
Configuring luci-app-aliyundrive-webdav.
Errors
uci: Entry not found

**Reply by messense** [reply]
可能不影响，登录进后台看看。

**Reply by askiter** [reply]
看着可以了，zh-cn的包也是这样，提示Entry not found，但是实际上安装上了
非常感谢
我试试看
